### PR TITLE
Promote buf convert to stable

### PR DESCRIPTION
**Pull request by joshcarp**
Promotes `buf beta convert` to stable because it's been out in beta for some time and there haven't been any bugs reported.
Keeps `buf beta convert` so that anyone who's using it may continue until we decide to remove it from beta.

### CODE REVIEW

File: private/buf/cmd/buf/buf.go [hunk 1]
```diff
@@ -140,7 +141,7 @@ func NewRootCommand(name string) *appcmd.Command {
 				Use:   "beta",
 				Short: "Beta commands. Unstable and likely to change.",
 				SubCommands: []*appcmd.Command{
-					convert.NewCommand("convert", builder),
+					//convert.NewCommand("convert", builder),
```
Comment (elliotmjackson) [inline private/buf/cmd/buf/buf.go:144]:
```suggestion
```

Comment (elliotmjackson) [inline private/buf/cmd/buf/buf.go:144]:
```suggestion
```

**Review by elliotmjackson — CHANGES_REQUESTED** [review]

File: private/buf/cmd/buf/buf.go (hunk 1 above)
Comment (elliotmjackson) [reply private/buf/cmd/buf/buf.go]: or do you want to uncomment it?

**Review by twilly — APPROVED** [review]
:shipit:

**Review by elliotmjackson — APPROVED** [review]

**Pull request merged**